tools/windows-node-installer/test/e2e/clustersetup.go

```diff
@@ -215,27 +216,27 @@ func deleteResources(kubeconfig string) error {
 	core, _ := coreclient.NewForConfig(cfg)
 
 	// delete role binding
-	if err := rbacConfig.RoleBindings(projectName).Delete(resourceName, &metav1.DeleteOptions{}); err != nil {
+	if err := rbacConfig.RoleBindings(projectName).Delete(context.TODO(), resourceName, metav1.DeleteOptions{}); err != nil {
```

Comment:
is there a reason for not passing a reference of &metav1.DeleteOptions{} to Delete anymore?